packages/components/psammead-navigation/src/ScrollableNavigation/index.jsx

```diff
@@ -1,15 +1,20 @@
 import React from 'react';
 import styled from '@emotion/styled';
-import { node, oneOf } from 'prop-types';
+import { node, oneOf, string } from 'prop-types';
 import { GEL_SPACING_SEXT } from '@bbc/gel-foundations/spacings';
 import {
   GEL_GROUP_2_SCREEN_WIDTH_MIN,
   GEL_GROUP_2_SCREEN_WIDTH_MAX,
 } from '@bbc/gel-foundations/breakpoints';
 
-/* Convert C_POSTBOX to rgba as IE doesn't like 8 digit hex */
-const C_POSTBOX_TRANSPARENT = `rgba(184, 0, 0, 0)`;
-const C_POSTBOX_OPAQUE = `rgba(184, 0, 0, 1)`;
+// Because IE11 can't handle 8-digit hex, need to convert to rgba
+const convertHexColourToDecimal = (hexColour, opacity = 1) => {
+  const hexChars = hexColour.split('');
+  const red = parseInt(hexChars[1], 16) * 16 + parseInt(hexChars[2], 16);
+  const green = parseInt(hexChars[3], 16) * 16 + parseInt(hexChars[4], 16);
+  const blue = parseInt(hexChars[5], 16) * 16 + parseInt(hexChars[6], 16);
+  return `rgba(${red}, ${green}, ${blue}, ${opacity})`;
+};
```

Comment:
This feels a bit dirty... so if anyone knows a nicer approach...

Reply:
The second answer reads a bit better I'd say: https://stackoverflow.com/questions/21646738/convert-hex-to-rgba

Reply:
Yeah it does, thanks.  I've updated with the best bits from there.